assignments/function-conversion.js

```diff
@@ -5,26 +5,56 @@
 // };
 // myFunction();
 
+let myFunction = () => {
+  console.log("Function was invoked!");
+};
+
+myFunction();
+
 // let anotherFunction = function (param) {
 //   return param;
 // };
 // anotherFunction("Example");
 
```

Comment:
is there a way to write another function on one line where the return is implicit vs explicit?

Reply:
let anotherFunction = (param) => param; <- is this what you mean where you can't tell what it is at first glance

antherFunction("Example");

Reply:
perfect